1-exercises/A-accessing-values/exercise2.js

```diff
@@ -7,22 +7,32 @@
 let capitalCities = {
   UnitedKingdom: "London",
   China: "Beijing",
-  Peru: "Lima"
+  Peru: "Lima",
```

Comment:
The last comma is optional

Reply:
It comes out after I save the code, maybe it is some VSCode extension.